frontend/public/components/chargeback.tsx

```diff
@@ -482,18 +467,17 @@ export type ReportDataState = {
   error: any,
   data: any,
   inFlight: boolean,
+  isReportFinished: boolean,
   sortBy: string,
   orderBy: string,
-  keys: string[],
+  cols: string[],
   rows: any[],
 };
 
 export type DataTableProps = {
   rows: any[],
-  orderBy: string,
-  sortBy: string,
-  applySort: any,
-  keys: string[],
+  cols: string[],
+  inFlight?: boolean,
```

Comment:
Is this field necessary ? dont see any occurrence in `DataTable `

Reply:
Nice catch!  yeah, not necessary, removed.